src/webapp/custom_ofmeet.js

```diff
@@ -681,6 +685,76 @@ var ofmeet = (function (ofm) {
         }
     }
 
+    //-------------------------------------------------------
+    //
+    //  setup welcome page
+    //
+    //-------------------------------------------------------
+
+    function setupWelcomePage() {
+		setupInprogressList();
+	}
+
+    function setupInprogressList() {
+        if (interfaceConfig.IN_PROGRESS_LIST_ENABLED) {
+            $('#react').on('click.tab-button', '.tab-buttons .tab', () => setTimeout(() => refreshInprogressListDOM(), 0));
+
+            if (interfaceConfig.IN_PROGRESS_LIST_INTERVAL > 0) {
+                inprogressListUpdateInterval = setInterval(() => updateInprogressList(), interfaceConfig.IN_PROGRESS_LIST_INTERVAL * 1000);
+            }
+            updateInprogressList();
+        }
+    }
+
+    function updateInprogressList() {
+        fetch("inProgressList.json")
```

Comment:
May we add something to this *fetch* request (parameter, header) to be evaluated in *InProgressListServlet.doGet()* to identify that it was send by the client of logged-in user?

I would like to hide the whole progress list (or at least the list of member, if enabled) from the "public". At the moment, there's just a "incomplete poor-man's protection" against crawlers (based on the referrer).

Reply:
Could please raise an issue for this, otherwise it will get lost 👍

Reply:
> May we add something to this _fetch_ request (parameter, header) to be evaluated in _InProgressListServlet.doGet()_ to identify that it was send by the client of logged-in user? 
> I would like to hide the whole progress list (or at least the list of member, if enabled) from the "public". At the moment, there's just a "incomplete poor-man's protection" against crawlers (based on the referrer).

There is nothing wrong with adding it, but I don't think it will be a drastic security measure.
This is because the parameters required for fetch are known by looking at the source code.

Reply:
I think about something that's a real proof for a valid user.